sympy/physics/control/lti.py

```diff
@@ -0,0 +1,93 @@
+from sympy import Basic, Mul, degree, Symbol, sympify, expand
+from sympy.core.numbers import Integer
+
+__all__ = ['TransferFunction',]
+
+
+class TransferFunction(Basic):
+
+    def __new__(cls, num, den, var):
+
+        if not isinstance(var, Symbol):
+            raise TypeError("Last argument must be a complex Symbol.")
+        if den == 0:
+            raise ValueError("TransferFunction can't have a zero denominator.")
+
+        num, den = sympify(num), sympify(den)
+        obj = Basic.__new__(cls, num, den, var)
```

Comment:
Why not use `super()` instead of calling via class's name?

Reply:
I haven't seen any class in sympy Codebase which uses `super()`. Every class which subclass from `Basic` uses `Basic.__new__(cls, ....)`

Reply:
OK, I will use `super(TransferFunction, cls).__new__(cls, num, den, var)`.